src/views/CVERecord/SearchResults.vue

```diff
@@ -0,0 +1,300 @@
+                            </span>
+                            <span v-if="typeof cveListSearchStore.pagination.toCount !== 'undefined'" class="has-text-weight-bold">
+                              -
+                              {{ cveListSearchStore.pagination.toCount.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',') }}
+                            </span>
+                          </span>
+                        </span>
+                        <span> of </span>
+                        <span class="has-text-weight-bold">
+                          {{ cveListSearchStore.totalSearchResultCount.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',') }}
+                        </span>
+                        <span> results for </span>
+                        <span class="has-text-weight-bold">{{ cveListSearchStore.query }}</span>
+                      </p>
+                    </div>
+                    <div class="field is-grouped is-grouped-right pb-3" style="border-bottom: 2px solid #e5e5e5;">
+                      <label class="label">Show:</label>
+                      <div class="control">
+                        <div class="select">
+                          <select v-model="cveListSearchStore.pagination.numberPerPage" aria-label="Select how many search results to show">
+                            <option v-for="num in cveListSearchStore.pagination.numberPerPageOptions" :value="num">{{ num }}</option>
+                          </select>
+                        </div>
+                      </div>
+                      <label for="sort" class="label">Sort by:</label>
+                      <div class="control">
+                        <div class="select">
+                          <select name="partner" v-model="cveListSearchStore.pagination.sortBy" aria-label="Select a sort option">
+                            <option v-for="sort in cveListSearchStore.pagination.sortByOptions" :key="sort.key"
+                              :value="sort"
+                            >
+                              {{ sort.label }}
+                            </option>
+                          </select>
+                        </div>
+                      </div>
+                    </div>
+                    <div v-if="Object.keys(cveListSearchStore.recordData).length > 0 || cveListSearchStore.totalSearchResultCount > 0">
+                      <div class="columns cve-columns" v-for="(result, resultIndex) in cveListSearchStore.searchResults" :key="result.index">
+                          <div class="column cve-column">
+                            <div class="column cve-column" :class="{'cve-top-border': resultIndex !== 0}">
+                            </div>
+                            <div class="columns cve-columns">
+                              <div class="column cve-column">
+                                <!-- <router-link :to="`/CVERecord?id=${result.cveId}`" target="_blank">{{ result.cveId }}</router-link> -->
```

Comment:
Can we delete this commented line?

Reply:
I uncommented the commented line because that's ultimately the link to use and instead, deleted the  sibling cve.org placeholder link.